src/views/Home.jsx

```diff
@@ -17,33 +18,70 @@ export function Home({ handleClick, setListToken }) {
 			navigate('/list');
 		} else {
 			setMessageError('Invalid token - no existing list');
+			setListToken(null);
+			setJointListToken('');
+			setTimeout(() => {
+				setMessageError('');
+			}, 3000);
 		}
 	};
 	const handleJointListTokenChange = (e) => {
 		setJointListToken(e.target.value);
 	};
 	return (
-		<div className="Home">
-			<button type="button" onClick={handleClick}>
-				Create list
-			</button>
-			<p>- or -</p>
-			<p>Join an existing shopping list by entering a three word token</p>
-			<form onSubmit={handleTokenSubmit}>
-				<label htmlFor="token">
-					<input
-						type="text"
-						id="token"
-						name="token"
-						placeholder="Three word token"
-						value={jointListToken}
-						onChange={handleJointListTokenChange}
-						required
-					/>
-				</label>
-				<p>{messageError}</p>
-				<button type="submit">Join an existing list</button>
-			</form>
+		<div className="Home flex flex-col items-center gap-5 w-full text-center mt-10 px-[5vw]">
+			<div>
+				<img
+					className="rounded-md block mb-5 max-w-full mx-auto"
+					src={welcomeImage}
+					alt="shopping cart with items and a person"
+				/>
+			</div>
+			<div>
+				<h1 className="text-4xl font-bold mb-5">
+					Welcome to Smart Shopping List!
+				</h1>
+
+				<p className="mb-5">
+					The app that learns how you shop. Smarter than your average shopping
+					list.
+				</p>
+			</div>
+			<div className="flex items-center justify-center flex-col text-center w-full">
+				<button
+					className="bg-[#008882] rounded-lg py-1 px-2 mb-7 w-full text-white font-medium"
+					area-label="create list"
+					type="button"
+					onClick={handleClick}
+				>
+					create a list
+				</button>
+				<p className="mb-7">
+					or enter a three word token to display an existing list
+				</p>
+				<form className="w-full" onSubmit={handleTokenSubmit}>
+					<label htmlFor="token">
```

Comment:
**Todo:** It looks like we had all the best intentions of adding a label here but forgot to actually add a label in the html element. If y'all don't want the label to display visually, you can opt for an `aria-label` instead! Here are some [docs](https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text)